superset-frontend/stylesheets/less/cosmo/variables.less

```diff
@@ -220,7 +220,7 @@
 );
 
 // ** `.form-group` margin
-@form-group-margin-bottom: 16px;
+@form-group-margin-bottom: 8px;
```

Comment:
we're going to be removing the .less styling, and also so that we don't change things too globally, you can add some styling for this selector with the emotion styles in `DatasourceEditor.jsx` under `const DatasourceContainer = styled.div`

Reply:
Hello, I was also uncomfortable changing this, since it is a global change. In the ticket, Stephen mentioned that he wanted the change for all form groups. So I thought that it might be for everything. Relooking at it, he might have meant for all of the form groups on this page however. 

Will go back and change it using emotion styles instead. It will be a good opportunity to work with emotion.

Reply:
Sorry for taking so long, Beto was going through the backend architecture. I just made this change.